pointpats/quadrat_statistics.py

```diff
@@ -210,9 +211,9 @@ def __init__(self, pp, lh):
         self.count_row_odd = int(math.ceil(range_y/(self.semi_height*2)))
 
         # quadrat number
-        self.num = self.count_row_odd * (self.count_column/2 +
+        self.num = self.count_row_odd * (int(self.count_column/2) +
```

Comment:
Oh goodness, that's definitely not necessary! Just a future reference kind of thing :smile: